src/scale.ts

```diff
@@ -433,14 +416,14 @@ export const defaultScaleConfig: ScaleConfig = {
 
 export interface SchemeParams {
   /**
-   * A color scheme name for sequential/ordinal scales (e.g., `"category10"` or `"viridis"`).
+   * A color scheme name for ordinal scales (e.g., `"category10"` or `"blues"`).
    *
    * For the full list of supported schemes, please refer to the [Vega Scheme](https://vega.github.io/vega/docs/schemes/#reference) reference.
    */
   name: string;
 
   /**
-   * For sequential and diverging schemes only, determines the extent of the color range to use. For example `[0.2, 1]` will rescale the color scheme such that color values in the range _[0, 0.2)_ are excluded from the scheme.
+   * For schemes only, determines the extent of the color range to use. For example `[0.2, 1]` will rescale the color scheme such that color values in the range _[0, 0.2)_ are excluded from the scheme.
```

Comment:
This is a property of a scheme param, so I'm not sure if the text "For schemes only," makes sense here.

Reply:
Fixed